Smash_Combos.Persistence/Migrations/20200816160856_UpdatedForModeration.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace Smash_Combos.Persistence.Migrations
+{
+    public partial class UpdatedForModeration : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "Admin",
+                table: "Users");
+
+            migrationBuilder.AddColumn<int>(
+                name: "UserType",
+                table: "Users",
+                nullable: false,
+                defaultValue: 1);
+
+            migrationBuilder.CreateTable(
+                name: "Infractions",
+                columns: table => new
+                {
+                    Id = table.Column<int>(nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.IdentityByDefaultColumn),
+                    UserId = table.Column<int>(nullable: false),
+                    ModeratorId = table.Column<int>(nullable: false),
+                    BanDuration = table.Column<int>(nullable: false),
+                    Points = table.Column<int>(nullable: false),
+                    Body = table.Column<string>(nullable: false),
+                    DateInfracted = table.Column<DateTime>(nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_Infractions", x => x.Id);
+                    table.ForeignKey(
+                        name: "FK_Infractions_Users_UserId",
```

Comment:
There also seems to be missing a constraint for moderator id

Reply:
I'm so used to migrations taking care of everything that I missed this and the other foreignkey missing. 😅 

I'll set the ReferentialAction for this constraint to SetNull, since we don't want bans to disappear if a moderator deletes their account.

Reply:
@sleeping-dev If you set this to SetNull, you need to make ModeratorId nullable.